src/NuGet.Indexing/ResponseFormatter.cs

```diff
@@ -346,7 +346,19 @@ public static void WriteStatsResult(JsonWriter jsonWriter, NuGetIndexSearcher se
             jsonWriter.WriteStartObject();
             WriteProperty(jsonWriter, "numDocs", searcher.IndexReader.NumDocs());
             WriteProperty(jsonWriter, "indexName", searcher.Manager.IndexName);
+            WriteProperty(jsonWriter, "machineName", searcher.MachineName);
             WriteProperty(jsonWriter, "lastReopen", searcher.LastReopen);
+            WriteProperty(jsonWriter, "lastIndexReloadTime", searcher.LastIndexReloadTime);
+            WriteProperty(jsonWriter, "lastIndexReloadDurationInMilliseconds", searcher.LastIndexReloadDurationInMilliseconds);
```

Comment:
I went as per @ryuyu's requirements in https://github.com/NuGet/NuGetGallery/issues/3551. If my understanding is correct, these values will be read by the runner which will ultimately expose these as perf counters to geneva. Peter can elaborate.